connectors/snmp-connector/clients/nediClient.go

```diff
@@ -271,3 +301,51 @@ func int2ip(val int) string {
 	binary.BigEndian.PutUint32(ip, uint32(val))
 	return ip.String()
 }
+
+func getMonitoredDevices(client *NediClient) (map[string]Monitoring, error) {
+	// TODO: remove this, use log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
Have you looked into log.SetFlags? Also, it would be more efficient to use log level checks like:

if (log level debug) {
     log.debug(....
}

We should create a JIRA to cleanup logging...

Reply:
Bad idea to have such check in Go. It's not Java. I think we have good logic for log level handling

Reply:
I don't agree with "bad idea". Let's push this back to next release and resolve then